Dockerfile

```diff
@@ -0,0 +1,14 @@
+ARG image_tag=latest
+
+FROM libero/browser_composer:${image_tag} AS composer
+FROM php:7.2.8-fpm-alpine3.7
```

Comment:
the base images sound good to me but agreed that they should be the subject of wider discussion @stephenwf @elrossco22

Reply:
Poll on Slack shows preference for these base images so far wrt Fedora and larger containers with nginx+fpm.